src/ensembl/src/content/app/browser/drawer/drawerActions.ts

```diff
@@ -1,16 +1,61 @@
 import { createAction } from 'typesafe-actions';
 
 import { getDrawerAnalyticsObject } from 'src/analyticsHelper';
+import { ActionCreator, Action, Dispatch } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+import { RootState } from 'src/store';
+import { getBrowserActiveGenomeId } from '../browserSelectors';
 
-export const changeDrawerView = createAction(
-  'drawer/change-drawer-view',
+export const changeDrawerViewForGenome = createAction(
+  'drawer/update-drawer-view',
   (resolve) => {
-    return (drawerView: string) =>
+    return (drawerView: { [genomeId: string]: string }) =>
       resolve(drawerView, getDrawerAnalyticsObject('User Interaction'));
```

Comment:
This is a useless analytics event.

Reply:
Why is this useless? Don't we want to know what drawer views users see the most?

Reply:
If I read the correctly, the only information that will be sent here to GA is category drawer, label  "User Interaction", which doesn't tell us much.

This analytics event is removed in Manoj's PR https://github.com/Ensembl/ensembl-client/pull/113/files